soco/core.py

```diff
@@ -238,6 +244,20 @@ def __str__(self):
     def __repr__(self):
         return '{0}("{1}")'.format(self.__class__.__name__, self.ip_address)
 
+    @classmethod
+    def any_soco(cls):
```

Comment:
Actually, it's not that easy, since it keys into the inheritance mechanism in the SoCo class (which we need to support different subclasses of SoCo for people who want them).  The discovery module does not know enough about all of this.  I'll think about how we might do it, but for the moment, I might leave this where it is